### PR TITLE
update gcb-docker-gcloud image to latest 1.20 version

### PR DESCRIPTION
**Pull request by humblec**
- [x] cloud build image to 1.20.10 which got many CVE fixes in place.

Signed-off-by: Humble Chirammal <humble.devassy@gmail.com>

### CODE REVIEW

**Comment by humblec** [comment]
cc @cpanato @saschagrunert

File: images/gcb-docker-gcloud/cloudbuild.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ steps:
     - gcr.io/$PROJECT_ID/gcb-docker-gcloud:latest
 substitutions:
   _GIT_TAG: '12345'
-  _GO_VERSION: 1.20.5
+  _GO_VERSION: 1.20.10
```
Comment (cpanato) [inline images/gcb-docker-gcloud/cloudbuild.yaml:17]: should we bump to 1.21?

cc @BenTheElder

Comment (humblec) [reply images/gcb-docker-gcloud/cloudbuild.yaml:17]: I think it's better to keep the latest `1.20` Version in place as most of the consumers are still on 1.20 version. However please let me know if we need to bump this to 1.21.   @cpanato

Comment (cpanato) [reply images/gcb-docker-gcloud/cloudbuild.yaml:17]: lets keep that for now

Comment (cpanato) [reply images/gcb-docker-gcloud/cloudbuild.yaml:17]: maybe we can have both configs? 
cc @BenTheElder @aojea

**Comment by humblec** [comment]
@aojea can you please approve this ?

**Comment by humblec** [comment]
@cpanato can you help to get approval here ? cc @saschagrunert

**Review by saschagrunert — APPROVED** [review]
I don't think I have approval rights here.

**Comment by humblec** [comment]
> I don't think I have approval rights here.

Thanks for the attempt @saschagrunert 👍

**Pull request merged**